lib/sendcloud/operations/parcels_request.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Sendcloud
+  module Operations
+    class ParcelsRequest < Operation
+      private
+
+      def handle_response_body(body)
+        body[:parcels]
```

Comment:
@gdackys there is an issue with this, in that we don't return the cursor information from the payload:

![image](https://user-images.githubusercontent.com/3620459/134928037-28a90814-f9c5-4bdc-9db7-c636d2d81660.png)

IMO we should either:

1) Return the `body` here as-is, so that it also contains the `next` and `previous` values, or
2) Handle pagination within the gem and return the array of all parcels

I'd prefer option 2 as it makes more sense IMO to have this handled by the carrier integration gem, then we don't need to worry about it in the backend - we can just make a request for parcels and have them all returned. All of the request logic is in one place, and if the behaviour of the API changes then we only need to fix this in the gem.

Reply:
If you are happy with option 2 I'll put in a follow-up card to add this to the gem, and carry on with the backend card as-is